R/zTest.R

```diff
@@ -609,18 +660,18 @@ designSafeZ <- function(meanDiffMin=NULL, beta=NULL, nPlan=NULL,
     names(nPlan) <- if (is.na(n2Plan)) "n1Plan" else c("n1Plan", "n2Plan")
   }
 
-  result <- list("nPlan"=nPlan, "parameter"=phiS, "esMin"=meanDiffMin, "alpha"=alpha, "beta"=beta,
-                 "alternative"=alternative, "h0"=h0, "testType"=testType, "paired"=paired,
-                 "sigma"=sigma, "kappa"=kappa, "ciValue"=ciValue,
-                 "ratio"=ratio, "pilot"=FALSE, "lowN"=NULL, "highN"=NULL, "call"=sys.call(),
-                 "timeStamp"=Sys.time())
+  result <- list("nPlan"=nPlan, "nPlanTwoSe"=nPlanTwoSe, "parameter"=phiS, "esMin"=meanDiffMin,
+                 "alpha"=alpha, "beta"=beta, "betaTwoSe"=betaTwoSe, "alternative"=alternative,
+                 "h0"=h0, "testType"=testType, "paired"=paired, "sigma"=sigma, "kappa"=kappa,
+                 "nPlanBatch"=nPlanBatch, "ratio"=ratio, "pilot"=FALSE,
+                 "logImpliedTarget"=logImpliedTarget, "logImpliedTargetTwoSe"=logImpliedTargetTwoSe,
+                 "lowN"=NULL, "highN"=NULL, "call"=sys.call(), "timeStamp"=Sys.time(),
+                 "note"=note, "bootObjN1Plan"=bootObjN1Plan, "bootObjBeta"=bootObjBeta,
```

Comment:
Why do you return the bootstrap result? Not needed for the print if I have understood correctly?

Reply:
They aren't needed, but I tend to keep it for debug purposes in a similar fashion jags/bugs and stan would output the mcmc samples.